src/geometry/manifold/manifold.lean

```diff
@@ -440,12 +438,17 @@ instance has_groupoid_model_space (H : Type*) [topological_space H] (G : structu
     simp [he, he', structure_groupoid.id_mem]
   end }
 
-variable [topological_space H]
+/-- Any manifold structure is compatible with the groupoid of all local homeomorphisms -/
+instance has_groupoid_continuous_groupoid : has_groupoid H M (continuous_groupoid H) :=
```

Comment:
I'm not sure if it would be better to use `\top` here... Maybe it doesn't really matter.

Reply:
I am not really expecting to use the lattice library here. I only defined the order because it is really natural, but otherwise it will not be important. I used `continuous_groupoid` instead of `\top` for readability purposes (I cannot expect that all readers remember the order structure and what `\top` is, but on the other hand `continuous_groupoid` is trivial to understand). But I can switch if you like `\top` better.

Reply:
I completely understand. And if you don't intend to use the lattice library, I think this is totally fine. Otherwise we would need to adhere to their syntax. So let's not bother with it now, and leave it as it is.